test/org/sagebionetworks/bridge/stormpath/StormpathAccountTest.java

```diff
@@ -22,11 +24,14 @@
 import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
 import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
 
+import com.newrelic.agent.deps.com.google.common.collect.Lists;
 import com.stormpath.sdk.account.Account;
 import com.stormpath.sdk.directory.CustomData;
 
 public class StormpathAccountTest {
```

Comment:
What I actually meant was a test where getActiveConsentSignature returns null because all of the consent signatures are withdrawn.